backend/src/main/java/com/github/clarakraus/susume/post/PostService.java

```diff
@@ -0,0 +1,23 @@
+package com.github.clarakraus.susume.post;
+
+import com.github.clarakraus.susume.blog.Movie;
+import com.github.clarakraus.susume.blog.MovieApiConnection;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class PostService {
+
+    private final PostRepo postRepo;
+    private final MovieApiConnection movieApiConnection;
+
+    public void createPost(Post post) {
+        post.setCategory(Category.Movie);
+        postRepo.save(post);
+    }
```

Comment:
could use checks for empty fields or any required fields. If certain fields need to be filled out and a user adds a post via Postman, some fields might be empty later on